docs/release_notes/0.0.14.md

```diff
@@ -1,6 +1,10 @@
 # Release 0.0.14
 
+This release makes it more fun to deploy applications
+
 ## Features
+* Adds a `okctl create application` command for scaffolding a configuration file which can be used to scaffold an application
+* Adds a `okctl apply application` command which will, based on the configuration file, scaffold kubernetes and argocd resources needed to deploy an application
```

Comment:
I'm not sure the user will understand what to "scaffold" means. Perhaps just say something like "based on the configuration file, "scaffolds kubernetes and argocd resource files" or something?

If we write simply "scaffold", we might scare the user, since s(he) doesn't know if any kind of magical things will be applied to the cluster.

The rest of the docs explains this, but it would be nice to not having to read the docs unless actually needed.

Reply:
Not sure I understand @yngvark 

Am I not saying exactly what you're suggesting on line 7? Should I use a synonym to "scaffold"?

Reply:
As mentioned in chat:
For instance: "scaffold kubernetes and argocd resources needed to deploy an application" -> "scaffold kubernetes and argocd resource files needed to deploy an application" - to explain that a file will be created, and not some resource in AWS for instance.